examples/rime-tsch/project-conf.h

```diff
@@ -85,4 +85,10 @@
 #undef TSCH_LOG_CONF_ID_FROM_LINKADDR
 #define TSCH_LOG_CONF_ID_FROM_LINKADDR(addr) ((addr) ? (addr)->u8[LINKADDR_SIZE - 2] : 0)
 
+#if CONTIKI_TARGET_COOJA
+#define COOJA_CONF_SIMULATE_TURNAROUND 0
+#define COOJA_CONF_TRANSMIT_ON_CCA 1
```

Comment:
This one is needed as well to run TSCH with Cooja...? As per `platform/cooja/dev/cooja-radio.h`, the default value of `COOJA_TRANSMIT_ON_CCA` is `1`... I may miss something.